### PR TITLE
Self service

### PR DESCRIPTION
**Pull request by harture**

### CODE REVIEW

File: api/account/api.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+// ConvertCredential creates an API credential from a KC credential
+func ConvertCredential(credKc *kc.CredentialRepresentation) CredentialRepresentation {
+	var cred CredentialRepresentation
+	cred.Id = credKc.Id
+	cred.Type = credKc.Type
+	cred.UserLabel = credKc.UserLabel
+	cred.CreatedDate = credKc.CreatedDate
+	cred.CredentialData = credKc.CredentialData
+	cred.Temporary = credKc.Temporary
```
Comment (bsoniam) [inline api/account/api.go]: how about cred.Value?

File: Gopkg.toml [hunk 2]
```diff
@@ -31,7 +31,8 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/keycloak-client"
-  version = "1.0-rc3"
+  branch = "credentials"
```
Comment (fperot74) [inline Gopkg.toml]: You really wanted to merge this file ?

File: configs/keycloak_bridge.yml [hunk 3]
```diff
@@ -68,6 +69,7 @@ db-audit-ro-max-idle-conns: 2
 db-audit-ro-conn-max-lifetime: 10
 db-audit-ro-migration: false
 db-audit-ro-migration-version: 0.1
+db-audit-ro-connection-check: false
```
Comment (fperot74) [inline configs/keycloak_bridge.yml]: This is just for our test environments, right ? We don't want to use this setting when deployed ?

Comment (harture) [reply configs/keycloak_bridge.yml]: This is just for our local dev env

File: Gopkg.toml (hunk 2 above)
Comment (harture) [reply Gopkg.toml]: Created PR (https://github.com/cloudtrust/keycloak-client/pull/18). Thanks to have notice it.

File: api/account/api.go (hunk 1 above)
Comment (harture) [reply api/account/api.go]: Value will be either used to transmit an info to Keycloak or to store a legacy secret. Thus we never want to return the value to final user.
I will add a comment about it as it not obvious at all.

Comment (harture) [reply api/account/api.go]: Value will be either used to transmit an info to Keycloak or to store a legacy secret. Thus we never want to return the value to final user.
I will add a comment about it as it not obvious at all.

**Pull request merged**